drivers/i2c/i2c_ll_stm32.c

```diff
@@ -424,3 +424,48 @@ static void i2c_stm32_irq_config_func_4(struct device *dev)
 #endif
 
 #endif /* CONFIG_I2C_4 */
+
+
+#ifdef CONFIG_I2C_5
```

Comment:
Conflict with https://github.com/zephyrproject-rtos/zephyr/pull/17818 to be taken care